setup.sh

```diff
@@ -20,7 +20,7 @@ echo "-----------------------------------------"
 if [ "$EUID" -eq 0 ]; then 
   echo "ERROR: Running this as root will cause permission issues."
   echo ""
-  echo "Please ensure your local user in in the docker group and run without sudo."
+  echo "Please ensure your local user is in the docker group and run without sudo."
```

Comment:
Wow, great catch. How long has that been there. 😊

Reply:
Lol, yes never noticed this until recently... didn't pick up on it before when I was looking for spelling errors, since the spelling is good! 🤣